### PR TITLE
WIP:SCB-2397 Add pack-dependencies module management dependencies

### PR DESCRIPTION
**Pull request by coolbeevip**
Optimized dependency management

### CODE REVIEW

File: pack-contracts/pack-contract-grpc/pom.xml [hunk 1]
```diff
@@ -41,6 +41,11 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-stub</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>3.19.2</version>
```
Comment (WillemJiang) [inline pack-contracts/pack-contract-grpc/pom.xml:47]: I think we need to specify the dependency version here with protobuf-java.version.

Comment (coolbeevip) [reply pack-contracts/pack-contract-grpc/pom.xml]: yes, I missed it here

Comment (coolbeevip) [reply pack-contracts/pack-contract-grpc/pom.xml]: done

**Comment by WillemJiang** [comment]
@coolbeevip  There are some error in alpha-fsm module when loading the spring boot application. Can you take a look at it?

**Comment by WillemJiang** [comment]
Create another PR for the work of SCB-2397.

**Pull request closed without merging**